plugin/buycourses/src/service_process.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/* For license terms, see /license.txt */
+/**
+ * Process payments for the Buy Courses plugin
+ * @package chamilo.plugin.buycourses
+ */
+/**
+ * Initialization
+ */
+require_once '../config.php';
+
+$currentUserId = api_get_user_id();
+
+if (empty($currentUserId)) {
+    header('Location: ' . api_get_path(WEB_CODE_PATH) . 'auth/inscription.php');
+    exit;
+}
+
+$em = Database::getManager();
+$plugin = BuyCoursesPlugin::create();
+$includeServices = $plugin->get('include_services');
+$paypalEnabled = $plugin->get('paypal_enable') === 'true';
+$transferEnabled = $plugin->get('transfer_enable') === 'true';
+
+if ($includeServices !== "true") {

```

Comment:
Better 'true' than "true" because the second one might get interpreted as the *value* true (as opposed to false).